components/Article/Page.js

```diff
@@ -65,10 +65,16 @@ import { withMarkAsReadMutation } from '../Notifications/enhancers'
 
 // Identifier-based dynamic components mapping
 import dynamic from 'next/dynamic'
-const Votebox = dynamic(() => import('../Vote/Voting'), {
+const dynamicOptions = {
   loading: () => <Loader />,
   ssr: false
-})
+}
+const Votebox = dynamic(() => import('../Vote/Voting'), dynamicOptions)
+const VoteCounter = dynamic(() => import('../Vote/VoteCounter'), dynamicOptions)
+const VoteResult = dynamic(
+  () => import('../Vote/VoteResultAuto'),
+  dynamicOptions
+)
```

Comment:
We should allow SSR for `VoteResult` because those are not personalised and can be cached:
```suggestion
const Votebox = dynamic(() => import('../Vote/Voting'), dynamicOptions)
const VoteCounter = dynamic(() => import('../Vote/VoteCounter'), dynamicOptions)
const VoteResult = dynamic(
  () => import('../Vote/VoteResultAuto'),
  { loading: () => <Loader /> }
)
```